### PR TITLE
Install "ftw.referencewidget".

### PR DESCRIPTION
**Pull request by Tschanzt**
Add ftw.referencewidget and use it instead of formwidget.contenttree for the contact field on the memberblock.

### CODE REVIEW

File: sources.cfg [hunk 1]
```diff
@@ -5,4 +5,9 @@ extends =
 extensions +=
     mr.developer
 
-auto-checkout += ftw.simplelayout
+auto-checkout +=
+    ftw.simplelayout
+    ftw.referencewidget
+
+[branches]
+ftw.referencewidget = tt-impl
```
Comment (Tschanzt) [inline sources.cfg]: 🚧

File: sources.cfg [hunk 2]
```diff
@@ -5,4 +5,9 @@ extends =
 extensions +=
     mr.developer
 
-auto-checkout += ftw.simplelayout
+auto-checkout +=
+    ftw.simplelayout
+    ftw.referencewidget
```
Comment (Tschanzt) [inline sources.cfg]: 🚧

**Comment by mbaechtold** [comment]
@Tschanzt, could you please update the changelog?

**Comment by Tschanzt** [comment]
@mbaechtold updated.

File: sources.cfg [hunk 3]
```diff
@@ -5,4 +5,5 @@ extends =
 extensions +=
     mr.developer
 
-auto-checkout += ftw.simplelayout
+auto-checkout +=
```
Comment (mbaechtold) [inline sources.cfg]: ✂️

**Comment by mbaechtold** [comment]
@Tschanzt, could you please squash the commits?

**Comment by Tschanzt** [comment]
@mbaechtold done

File: setup.py [hunk 4]
```diff
@@ -18,6 +18,8 @@
     ]
 simplelayout = [
     'ftw.simplelayout [contenttypes]',
+    'ftw.referencewidget',
+
```
Comment (mbaechtold) [inline setup.py]: ✂️

File: sources.cfg [hunk 5]
```diff
@@ -6,3 +6,4 @@ extensions +=
     mr.developer
 
 auto-checkout += ftw.simplelayout
+
```
Comment (mbaechtold) [inline sources.cfg]: ✂️

**Comment by Tschanzt** [comment]
@mbaechtold updated.

**Comment by mbaechtold** [comment]
@maethu, this looks good to me. What do you think?

**Comment by maethu** [comment]
👍 - Please make a new minor release

**Comment by mbaechtold** [comment]
New release is on the way...

**Pull request merged**